test/functional/eval/msgpack_functions_spec.lua

```diff
@@ -337,7 +337,8 @@ describe('msgpack*() functions', function()
     eq(1, eval('type(parsed[0]) == type(0) ' ..
                '|| parsed[0]._TYPE is v:msgpack_types.integer'))
     if eval('type(parsed[0]) == type(0)') == 1 then
-      eq(1, eval('0xFFFFFFFFFFFFFFFF == parsed[0]'))
+      command('call assert_equal(0xFFFFFFFFFFFFFFFF, parsed[0])')
```

Comment:
And this looks overly strange: UINT64_MAX is *unsigned* integer maximum, it *must not* fit into one `varnumber_T` if it is `int64_t` and not a 128-bit integer available on some systems. If test fails because it entered this branch then there is a bug in `msgpackparse()`.